packages/editor-ui/src/components/executions/workflow/VoteButtons.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+	<div :class="$style.ratingIcon">
+		<n8n-icon-button
+			:class="{ [$style.up]: vote === 'up' }"
+			type="tertiary"
+			text
+			size="medium"
+			icon="thumbs-up"
+			@click="onVoteClick('up')"
+		/>
+		<n8n-icon-button
+			:class="{ [$style.down]: vote === 'down' }"
+			type="tertiary"
+			text
+			size="medium"
+			icon="thumbs-down"
+			@click="onVoteClick('down')"
+		/>
+	</div>
+</template>
+
+<script setup lang="ts">
+import { defineProps, defineEmits } from 'vue';
+import type { AnnotationVote } from 'n8n-workflow';
+
+defineProps<{
+	vote: AnnotationVote | null;
+}>();
+
+const emit = defineEmits<{
+	'vote-click': [vote: AnnotationVote];
+}>();
+
+const onVoteClick = (vote: AnnotationVote) => {
+	emit('vote-click', vote);
+};
+</script>
+
+<style module lang="scss">
+.ratingIcon {
```

Comment:
A small bug I encountered: there's some focus state on these buttons, so after upvoting/downvoting the color is always red. This is confusing when you're trying to undo the downvote as it appears red:

https://github.com/user-attachments/assets/819f262f-f708-497d-b25a-fa901f866a4e

Reply:
Can you please advise on how to better fix that? I suppose I need to override some styles of N8nButton, but not sure which one